test/decoders/test_video_decoder.py

```diff
@@ -249,7 +249,7 @@ def test_getitem_slice(self, device, seek_mode):
             ]
         )
         for sliced, ref in zip(all_frames, decoder):
-            if not (in_fbcode() and device == "cuda"):
```

Comment:
Here and below, it turns out the "internal" problems we had were because of FFmpeg4. We couldn't tell at the time, because we had no FFmpeg4 CUDA CI running on GitHub.